dev/src/convert.ts

```diff
@@ -177,16 +177,12 @@ export function valueFromJson(fieldValue: api.IValue): api.IValue {
       return {
         bytesValue: bytesFromJson(fieldValue.bytesValue!),
       };
-    case 'integerValue':
```

Comment:
I dropped this since the client supports "string|number" throughout. Converting to Number results in precision loss.